src/services/fetchCarousel.js

```diff
@@ -1,15 +1,23 @@
-
 async function fetchCarousel(setCarouselItems, carouselCategory) {
-	try {
-		const response = await fetch(`/api/v1/${carouselCategory}`);
-		if (!response.ok) {
-			throw new Error(`HTTP error! Status: ${response.status}`);
-		}
-		const data = await response.json();
-		setCarouselItems(data.carouselItems);
-	} catch (err) {
-		console.log('Error: ', err.message);
-	}
+  try {
+    const { data } = await axios.get(`/api/v1/${carouselCategory}`)
```

Comment:
missing import for axios, 
this makes application crashed when trying to render carousel component

Reply:
ok